prawnto.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
 
   s.required_rubygems_version = ">= 1.3.6"
   s.platform = Gem::Platform::RUBY
-  s.add_dependency('rails', '>= 3.2')
```

Comment:
@C-Flatla I think this line forces me to have rails 5.2 running. Is this as expected?